Sdk/Speckle.Connectors.Utils/ContainerRegistration.cs

```diff
@@ -14,6 +15,8 @@ public static void AddConnectorUtils(this SpeckleContainerBuilder builder)
   {
     // send operation and dependencies
     builder.AddSingleton<CancellationManager>();
+    builder.AddScoped<RootObjectUnpacker>();
+    builder.AddScoped<RootObjectUnpackerResult>();
```

Comment:
Is this right? Why would we need to register a scoped `RootObjectUnpackerResult`?`

Reply:
We need them only on Receive operation. `RootObjectUnpackerResult` is just record. How would you do it?

Reply:
i've removed